.pre-commit-config.yaml

```diff
@@ -24,19 +24,19 @@ repos:
   #
   # Built-ins
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.5.0
```

Comment:
Have you tested these changes in MT?

Reply:
this based on while running locally but I did run make test and make check . Not sure how to check it in MT

Reply:
ptal: https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/blob/main/README.md

Reply:
@Ankit152  Could you be precise on how to go about the testing , the read me seems to have lot of information .

Reply:
You could try building the images `locally` in MTB and also test the particular `tasks` present in `MT` repo.

Reply:
@Ankit152 I have updated the ticket https://issues.redhat.com/browse/MTSRE-693 with the output of building images locally in MTB . On running the tasks lets sync .

Reply:
Looks good to me.